README.md

```diff
@@ -34,24 +168,17 @@ scrape_configs:
 
   # The job name is added as a label `job=<job_name>` to any timeseries scraped from this config.
   - job_name: 'maia'
-    metrics_path: "/api/v1/metrics"
+    metrics_path: "/federate"
```

Comment:
Why is `/federate` not strictly part of the API? The behavior of this endpoint doesn't differ from the others: Maia accepts requests with basic-auth, adds the scope identifier to the subsequent req. to prometheus and returns the results. So, as for the other methods, I'd say it's part of the API.